content/review.rst

```diff
@@ -201,9 +201,24 @@ A merge commit should have a commit message, in the format::
   commit to a ticket / branch / review / task and check the history/story of
   that commit.
 
+A note on Trac review requests
+==============================
+
+* When submitting a Trac ticket for review, please bear in mind that this is
+  the moment to add relevant documentation you updated, affected repos, etc.
+  
+* A review in one of your Trac tickets is not just to say "please look through
+  this issue to see that I worked so I can close it", but instead a "I have
```

Comment:
But this template does not appear on a Trac. My intention was to make it clearer how Trac reviews should be treated and remove any pre-conceptions from using other ticketing systems. May be second nature to you as you used it for so long, or maybe it is just me that saw it that way. 
We may just clarify this by adding something like "All of the above applies for reviews through Github, email, Trac (even for Trac tickets not associated with Github)"